search-app/src/cmr/search/services/query_service.clj

```diff
@@ -272,11 +272,14 @@
   (when-let [[start-date end-date] (mapv time-format/parse
                                          (string/split (:has-granules-created-at params) #","))]
     (let [query (qm/query {:concept-type :granule
-                           :condition (qm/date-range-condition
-                                       :created-at start-date end-date)
-                           :page-size :unlimited
+                           :page-size 0
                            :result-format :query-specified
-                           :result-fields [:collection-concept-id]})]
+                           :condition (qm/date-range-condition
+                                        :created-at start-date end-date)
+                           :result-fields []
+                           :aggregations {:collections
+                                          {:terms {:size 50000
```

Comment:
Let's define the 50000 as a var (e.g. collection-aggregation-size) so we can better track it.